torchquantum/measurement/measurements.py

```diff
@@ -23,7 +23,6 @@
     "expval",
     "MeasureAll",
     "MeasureMultipleTimes",
-    "MeasureMultiPauliSum",
```

Comment:
Because it was not correctly implemented. You can check the example pauli_sum_op.py, changing the coefficients has no effect on the outcome of the measurement (see my first post). The coefficient part of the input is not taken into account.

Thinking about it, I think we could implement this easily by using 
https://github.com/mit-han-lab/torchquantum/blob/611cc2ac9d5d7490114ec4ca175c2b74e0951f38/torchquantum/algorithm/hamiltonian.py#L93-L112
and using the matrix of the hamiltonian.

Alternatively, we could change `expval_joint_analytical` to not only take the string of observables, but also coefficient info and multiply the cofficients at the point where the kronecker product is taken here:
https://github.com/mit-han-lab/torchquantum/blob/611cc2ac9d5d7490114ec4ca175c2b74e0951f38/torchquantum/measurement/measurements.py#L270-L271